test/functional/wallet_abandonconflict.py

```
@@ -104,25 +110,25 @@ def run_test(self):
         # inputs are still spent, but change not received
-        newbalance = self.nodes[0].getbalance()
+        newbalance = alice.getbalance()
         assert_equal(newbalance, balance - signed3_change)
         # Unconfirmed received funds that are not in mempool, also shouldn't show
         # up in unconfirmed balance
-        balances = self.nodes[0].getbalances()['mine']
+        balances = alice.getbalances()['mine']
         assert_equal(balances['untrusted_pending'] + balances['trusted'], newbalance)
         # Also shouldn't show up in listunspent
-        assert not txABC2 in [utxo["txid"] for utxo in self.nodes[0].listunspent(0)]
+        assert not txABC2 in [utxo["txid"] for utxo in alice.listunspent(0)]
```

Ruff (0.12.2)

Test for membership should be `not in`

Convert to `not in`

(E713)